api/tests/test_conf.rs

```diff
@@ -0,0 +1,120 @@
+#[cfg(test)]
+mod tests {
+    use boom_api::conf::{load_config, load_dotenv};
+    use std::panic;
+
+    #[test]
+    fn test_token_expiration_validation_fails_with_zero() {
+        load_dotenv();
+
+        // This should panic because token_expiration is 0
+        let result = panic::catch_unwind(|| {
```

Comment:
Seems more complicated then I though, any reason to wrap the whole thing in this `panic` statement, including the string definition and such?